IaC/pxc.cd/init.groovy.d/cloud.groovy

```diff
@@ -158,120 +162,103 @@ initMap['micro-amazon'] = '''
             sudo mount ${DEVICE} /mnt
         fi
     fi
-    until sudo yum makecache; do
-        sleep 1
-        echo try again
-    done
-    sudo yum -y install java-1.8.0-openjdk git || :
-    sudo yum -y install aws-cli || :
-    sudo yum -y remove java-1.7.0-openjdk || :
-    sudo install -o $(id -u -n) -g $(id -g -n) -d /mnt/jenkins
-'''
-initMap['min-centos-6-x64'] = '''
-    set -o xtrace
-    if ! mountpoint -q /mnt; then
-        for DEVICE_NAME in $(lsblk -ndpbo NAME,SIZE | sort -n -r | awk '{print $1}'); do
-            if ! grep -qs "${DEVICE_NAME}" /proc/mounts; then
-                DEVICE="${DEVICE_NAME}"
-                break
-            fi
+    if [[ ${RHVER} -eq 6 ]]; then
+        sudo curl https://jenkins.percona.com/downloads/cent6/centos6-eol.repo --output /etc/yum.repos.d/CentOS-Base.repo
+        until sudo yum makecache; do
+            sleep 1
+            echo try again
         done
-        if [ -n "${DEVICE}" ]; then
-            sudo mkfs.ext2 ${DEVICE}
-            sudo mount ${DEVICE} /mnt
+        if [[ ${ARCH} == "x86_64" ]]; then
+            PKGLIST="epel-release centos-release-scl"
+        else
+            PKGLIST="epel-release"
+            until sudo yum -y update; do
+                sleep 1
```

Comment:
why we perform `yum update` without installing epel repo? maybe some updates would come from it?

Reply:
The same way was with PXB
Each centos-6-x32 base image is outdated, and when we install EPEL-release before any updates, it will fail to connect to the repos with SSL error

Update is needed to perform makecache after it